src/containers/Validators/simpleTab.scss

```diff
@@ -1,141 +1,21 @@
 @import '../shared/css/variables';
 
 .simple-body {
-  border-top: 1px solid $gray-800;
-
-  .index .val img {
-    height: 16px;
-    padding-top: 2px;
-  }
-
   .rows {
-    width: 100%;
-    padding-right: 0px;
-    padding-bottom: 40px;
-    float: left;
-
-    @include for-size(desktop-up) {
-      width: calc(100% - 276px - 32px);
-      padding-bottom: 60px;
-      border-right: 1px solid $gray-800;
-    }
-
     .row {
-      display: flex;
-      overflow: hidden;
-      padding: 20px 10px;
-      border-bottom: 1px solid $gray-800;
-
-      @include for-size(tablet-portrait-up) {
-        padding: 24px;
-      }
-
       &.flex-wrap {
         flex-wrap: wrap;
       }
 
       .label {
         display: flex;
-        flex-grow: 1;
         align-items: center;
-        color: $white;
-        font-size: 12px;
-        line-height: 20px;
-        text-transform: uppercase;
-        @include medium;
-
-        &.unauthorize {
-          color: #944;
-        }
-      }
-
-      .account {
-        display: inline-block;
-        overflow: hidden;
-        font-size: 12px;
-        line-height: 20px;
-        text-overflow: ellipsis;
-        vertical-align: middle;
-        white-space: nowrap;
       }
 
       .value {
-        flex-grow: 1;
-        padding-left: 5px;
-        color: $white;
-        float: right;
-        text-align: right;
-        vertical-align: middle;
-
         img {
           height: 16px;
         }
-
-        @include medium;
-
-        .currency {
```

Comment:
Line 76 - 127 do not apply to validators and must have been copied originally from `Transactions/simpleTab.scss`